crates/store/re_protos/src/v0/rerun.log_msg.v0.rs

```diff
@@ -0,0 +1,388 @@
+// This file is @generated by prost-build.
```

Comment:
Then is should be marked as `linguist-generated=true` in a `.gitattributes` file. It would be great if that file was generated/maintained by `build.rs` so it keeps up-to-date

Reply:
We could probably mark the whole folder as `linguist-generated=true`

Reply:
![image](https://github.com/user-attachments/assets/6e26f2cc-2835-4ce5-b273-eb1e54564b08)